openedx_proversity_reports/api/v0/views.py

```diff
@@ -13,18 +15,15 @@
 
 from openedx_proversity_reports.edxapp_wrapper.get_edx_rest_framework_extensions import get_jwt_authentication
 from openedx_proversity_reports.edxapp_wrapper.get_openedx_permissions import get_staff_or_owner
-from openedx_proversity_reports.tasks import (
-    generate_completion_report,
-    generate_last_page_accessed_report,
-    generate_time_spent_report,
-)
 
 logger = logging.getLogger(__name__)
+BLOCK_DEFAULT_REPORT_FILTER = ['vertical']
+SUPPORTED_TASKS_MODULE = 'openedx_proversity_reports.tasks'
```

Comment:
If the module task will be configurable, let's put it on the plugin settings. @andrey-canon

Reply:
This is not configurable, it is a simple constant

Reply:
Could it be configurable? @andrey-canon

Reply:
No, that is out of the scope

Reply:
In the future? just asking... @andrey-canon

Reply:
imo that is not necessary